documentation/concepts/data-model.rst

```diff
@@ -0,0 +1,71 @@
+.. _datamodel:
+
+The FlexMeasures data model 
+=============================
+
+The data model being used in FlexMeasures is visualized here (click for larger version):
+
+.. image:: https://raw.githubusercontent.com/FlexMeasures/screenshots/main/architecture/FlexMeasures-NewDataModel.png
+    :target: https://raw.githubusercontent.com/FlexMeasures/screenshots/main/architecture/FlexMeasures-NewDataModel.png
+    :align: center
+..    :scale: 40%
+
+
+Let's dive into some of the more crucial model types:
+
+
+Assets
+---------
+
+An asset is anything on which you collect data, e.g. a car battery or an industrial machine.
```

Comment:
Maybe adding here that `Assets` can represents physical or "virtual" objects might help. I feel that linking `Assets` to the beliefs is important but I think we need to emphasize that they represent "objects".